modules/Emails/include/ListView/ListViewHeader.js

```diff
@@ -37,18 +37,88 @@
  * display the words  "Powered by SugarCRM" and "Supercharged by SuiteCRM".
  */
 
+/**
+ * @todo take this function to a common email listview object
+ * @param {string} actionUrl
+ * @param {function} successCallback
+ * @param {function} errorCallback
+ * @param {string} loadingTitle
+ * @param {string} errorMessage
+ */
+var handleSelectedEmailsListViewItems = function(actionUrl, successCallback, errorCallback, loadingTitle, errorMessage) {
```

Comment:
Should we put this into a jQuery plugin? Perhaps in a simaler fashion to click() or focus() ?